Ball of Duty Server/Program.cs

```diff
@@ -13,25 +14,63 @@ namespace Ball_of_Duty_Server
 {
     internal class Program
     {
+        public enum Command
+        {
+            QUIT,
+            WRITE_ACTIVE_PLAYERS,
+            BAN,
+            SERVER_MESSAGE
+        }
+
+
+        public static Command? GetCommandType(String line)
+        {
+            if (line.ToLower().StartsWith("sm") || line.ToLower().StartsWith("server_message"))
+            {
+                return Command.SERVER_MESSAGE;
+            }
+            else if (line.ToLower().StartsWith("quit"))
+            {
+                return Command.QUIT;
+            }
+            else if (line.ToLower().StartsWith("active_players"))
+            {
+                return Command.WRITE_ACTIVE_PLAYERS;
+            }
+            else if (line.ToLower().StartsWith("ban"))
+            {
+                return Command.BAN;
+            }
+            else
+            {
+                return null;
+            }
+        }
+
+        public static string GetCommand(String line)
+        {
+            return line.IndexOf(" ") < 0 ? "" : line.Substring(line.IndexOf(" ") + 1);
```

Comment:
We dont want to split for each space, we want the whole string after the first word.

Ie GetCommand("ban player with id 12") would return "player with id 12". not "player", "with", "id", "12"